<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreUI</class>
 <widget class="QMainWindow" name="CoreUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - Core</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="logGroupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>321</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>系统日志</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QTextEdit" name="logTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>321</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="realTimeCaptureGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>摄像头实时画面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="realTimeCaptureLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>640</width>
       <height>451</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>&lt;font color=red&gt;摄像头未开启&lt;/font&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>进入系统</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
